src/optimize/base_optimizer.js

```diff
@@ -1,19 +1,23 @@
+import { resolve } from 'path';
+import { writeFile } from 'fs';
+import { inherits } from 'util';
```

Comment:
`inherits` and `formatUrl` are unused, `inherits` was unused before your changes but `formatUrl` was added